test/PublishCommand.js

```diff
@@ -536,12 +536,31 @@ describe("PublishCommand", () => {
             throw new Error(fs.readFileSync(path.join(testDir, "lerna-debug.log"), "utf8"));
           }
 
+          const directory = path.join(testDir, "./packages/package-2");
+          const packageName = path.basename(directory);
+
           expect(publishedTagInDirectories(testDir))
-            .toMatchSnapshot("[normal --skip-temp-tag] npm publish --tag");
+            .toMatchSnapshot("[normal --temp-tag] npm publish --tag");
 
-          expect(NpmUtilities.checkDistTag).not.toBeCalled();
-          expect(NpmUtilities.removeDistTag).not.toBeCalled();
-          expect(NpmUtilities.addDistTag).not.toBeCalled();
+          expect(NpmUtilities.checkDistTag).lastCalledWith(
```

Comment:
These `lastCalledWith()`s should be replaced by a `expect(addedDistTagInDirectories(testDir)).toMatchSnapshot("[normal --use-temp-tag] npm dist-tag add");` assertion.